### PR TITLE
Update dg find command

### PR DESCRIPTION
**Pull request by houtenteo**
Added activity diagram for the add command as well.
Also i fix the sequence diagram for the add command. Realised there were some mistakes

### CODE REVIEW

File: docs/diagrams/FindSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+participant CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant Model MODEL_COLOR
+participant "p:NameContainsKeywordPredicate" as namePredicate MODEL_COLOR
+participant "c:CombinedPredicate" as combinedPredicate MODEL_COLOR
+end box
+
+-> LogicManager : execute("findAny n/alex g/m")
```
Comment (Kangxun) [inline docs/diagrams/FindSequenceDiagram.puml:23]: is `"findAny n/alex g/m"` supposed to have `g/m`?

**Review by Kangxun — COMMENTED** [review]
looks pretty good just some edits

File: docs/diagrams/FindSequenceDiagram.puml (hunk 1 above)
Comment (houtenteo) [reply docs/diagrams/FindSequenceDiagram.puml:23]: woops

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -203,6 +212,48 @@ On the other hand, `Address`, `Gender`, `Age` and `Interest` are seen as complem
...
+The find commands are facilitated by the LogicManager.
+
+**How the find command is executed:**
+
+1. Command entered by user is passed into the LogicManager (ie. `findAny n/alex g/m` or `findAll n/alice g/f`)
+2. AddressBookParser parses the command
+3. AddressBookParser creates a FindAny/FindAll command with the respective predicates depending on the fields specified by the user
+4. LogicManager executes the Find command and the model updates the filtered person list with the new predicates
+
+The Sequence Diagram below illustrates the interactions within the Logic component for the `execute("n/alex g/m")` API call.
```
[the remark this answers is not in the document]
Comment (houtenteo) [reply docs/DeveloperGuide.md:226]: yes it is XD

File: docs/diagrams/AddActivityDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+:User enters add command;
+:AddressBookParser parses the command;
+:AddCommandParser parses the arguments provided;
+if () then (command is valid)
```
[the remark this answers is not in the document]
Comment (houtenteo) [reply docs/diagrams/AddActivityDiagram.puml:8]: right

File: docs/diagrams/FindActivityDiagram.puml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+'https://plantuml.com/activity-diagram-beta
+
+start
+:User enters findAny command;
+:AddressBookParser parses the command;
+:FindAnyCommandParser parses the arguments provided;
+if () then (command is valid)
+  :new predicates for each specified field are created;
+  :predicates are saved in a CombinedPredicate object;
+  :filtered list is updated with the new CombinedPredicate;
```
[the remark this answers is not in the document]
Comment (houtenteo) [reply docs/diagrams/FindActivityDiagram.puml:11]: idt so, i didnt really mention the creation of a findAnyCommand object

[the remark this answers is not in the document]
Comment (houtenteo) [reply docs/diagrams/FindActivityDiagram.puml:11]: maybe ill add the creation of a FindAnyCommand

**Review by Kangxun — APPROVED** [review]
LGTM

**Pull request merged**